norse/torch/module/receptive_field.py

```diff
@@ -112,7 +114,9 @@ def _update_weights(self):
                         in_weights = empty_weights.clone()
                         in_weights[i] = fields
                         weights.append(in_weights)
-                    self.weights = torch.concat(weights, 1).permute(1, 0, 2, 3)
+                    self.register_buffer(
```

Comment:
@dimitriskor, I just realized that buffers are not considered a model parameter. Should this have been `register_parameter`? https://pytorch.org/docs/stable/generated/torch.nn.Module.html#torch.nn.Module

Reply:
The decision for declaring the weights as buffers has been deliberate, as they are only part of the computational graph and not leafs.
The grad is set to True and the gradient computation flows backwards to the rf parameters which are the leaf nodes of the computational graph.